Big-Spender/readme.md

```diff
@@ -68,39 +68,39 @@ INSERT YOUR QUERY HERE
 **You:** Then here's the query for that:
 
 ```sql
-INSERT YOUR QUERY HERE
+SELECT * FROM spends WHERE description ILIKE '%fee%' OR description ILIKE '%fees%';
```

Comment:
so if you are using `%fee%`, doesn't this already cover the `'FEES'` use case? 

nice use of` ilike`.